packages/perseus/src/validation.types.ts

```diff
@@ -228,35 +229,35 @@ export type PerseusSorterUserInput = {
     changed: boolean;
 };
 
-export type PerseusTableRubric = {
+export type PerseusTableScoringData = {
     // Translatable Text; A 2-dimensional array of text to populate the table with
     answers: ReadonlyArray<ReadonlyArray<string>>;
 };
 
 export type PerseusTableUserInput = ReadonlyArray<ReadonlyArray<string>>;
 
-export type Rubric =
+export type ScoringData =
     | PerseusCategorizerScoringData
     | PerseusCSProgramRubric
```

Comment:
There's some types here that haven't been renamed from *Rubric to *ScoringData. Is there a reason for that?

Reply:
Oh, yes, these are changed in one of my other PRs